client/src/app/site/motions/services/motion-multiselect.service.ts

```diff
@@ -126,38 +130,27 @@ export class MotionMultiselectService {
      * @param motions The motions to change
      */
     public async setRecommendation(motions: ViewMotion[]): Promise<void> {
-        if (motions.every(motion => motion.workflow_id === motions[0].workflow_id)) {
-            const title = this.translate.instant(
-                'This will set the following recommendation for all selected motions:'
-            );
-
-            // hacks custom Displayables from recommendations
-            // TODO: Recommendations should be an own class
-            const choices: Displayable[] = this.workflowRepo
-                .getWorkflowStatesForMotions(motions)
-                .filter(workflowState => !!workflowState.recommendation_label)
-                .map(workflowState => ({
-                    id: workflowState.id,
-                    getTitle: () => workflowState.recommendation_label,
-                    getListTitle: () => workflowState.recommendation_label
-                }));
-            const clearChoice = this.translate.instant('Delete recommendation');
-            const selectedChoice = await this.choiceService.open(title, choices, false, null, clearChoice);
-            if (selectedChoice) {
-                const requestData = motions.map(motion => ({
-                    id: motion.id,
-                    recommendation: selectedChoice.action ? 0 : (selectedChoice.items as number)
-                }));
-                const message = `${motions.length} ` + this.translate.instant(this.messageForSpinner);
-                this.overlayService.showSpinner(message, true);
-                // await this.httpService.post('/rest/motions/motion/manage_multiple_recommendation/', {
-                //     motions: requestData
-                // });
-                throw new Error('TODO');
-            }
-        } else {
-            throw new Error(
-                this.translate.instant('You cannot change the recommendation of motions in different workflows!')
+        // hacks custom Displayables from recommendations
+        // TODO: Recommendations should be an own class
+        const title = this.translate.instant('This will set the following recommendation for all selected motions:');
+        const choices: Displayable[] = this.workflowRepo
+            .getWorkflowStatesForMotions(motions)
+            .filter(workflowState => !!workflowState.recommendation_label)
+            .map(workflowState => ({
```

Comment:
Why do you map this, it should not be needed? Recommendation is the name for a state with a recommendation_label; it is not a different model. (Same as the wording `amendment` for a motion with a lead motion assigned). If there are problems, please give me a call or so, so we can speak about it.